packages/integration-karma/test/profiler/sanity/profiler.spec.js

```diff
@@ -49,6 +49,18 @@ describe('Profiler Sanity Test', () => {
         Stop: 1,
     };
 
+    const RenderMode = {
+        Light: 0,
+        Shadow: 1,
+    };
+
+    const ShadowMode = {
+        Native: 0,
+        Synthetic: 1,
+    };
```

Comment:
This test shows how you can use this API. RenderMode/ShadowMode are expressed as 0 or 1.

It is very unlikely that these will change (we would need to change the order in our enum), and if they do change, this test will break.